applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -108,6 +121,68 @@ static struct bt_audio_discover_params audio_source_discover_param[CONFIG_BT_MAX
 static int discover_source(struct bt_conn *conn);
 #endif /* CONFIG_STREAM_BIDIRECTIONAL */
 
+/**
+ * @brief  Get the presentation delay for all headsets
+ *
+ * @param  mode   The mode of search for the presentation delay
+ * @param  index  The index of the headset to test against
+ *
+ * @return The chosen presentation delay in micro seconds or 0 if error
+ */
+static uint32_t find_headset_pres_delay(enum le_audio_pres_search mode, uint8_t index)
```

Comment:
The naming usually has the verb last. Suggest: 
```suggestion
static uint32_t headset_pres_delay_find(enum le_audio_pres_search mode, uint8_t index)
```